lib/ex_doc/formatter/html/autolink.ex

```diff
@@ -321,17 +321,23 @@ defmodule ExDoc.Formatter.HTML.Autolink do
     end
   end
 
+  @placeholder "ó"
```

Comment:
Let’s use $. But then we will need to escape it in the regex.

Reply:
Good call. I got some failing tests on CI, which I didn’t get locally, so there might still be something wrong.

Reply:
Never mind. It has to be a valid identifier which is why ó fails on v1.7. We will have to make it something longer then. A mixture of lowercase, uppercase, underscore and digits.